xarray/tutorial.py

```diff
@@ -19,13 +21,175 @@
 from .core.dataarray import DataArray
 from .core.dataset import Dataset
 
-_cache_name = "xarray_tutorial_data"
-_cache_dir = pathlib.Path.home() / ".cache"
-# TODO: I/O on import. Might not be a good idea.
-if _cache_dir.is_dir():
-    _default_cache_dir = _cache_dir / _cache_name
+# vendored from appdirs
```

Comment:
We still need to keep the appdirs copyright notice. The simplest way to do this is to keep appdirs in a separate, self-contained file, e.g., like what's done for Numba: https://github.com/numba/numba/blob/13ece9b97e6f01f750e870347f231282325f60c3/numba/misc/appdirs.py

Reply:
done, although the directory could have a better name